### ISSUE TITLE
Feature Request: Highlight Similarity to Selected Sequence

### ISSUE THREAD

**Issue by DDashevsky** [AliView/AliView#29]
I'm imagining the inverse of the currently available "Highlight diff from selected sequence" where it highlights bases/residues that are the same rather than different.

**Comment by ewallace** [comment]
Yes please! This would be very helpful for the case where one wants to look for patches of conservation.
